.prettierrc

```diff
@@ -1,6 +1,4 @@
 {
     "singleQuote": true,
-    "tabWidth": 4,
```

Comment:
We do want to use tab width of 2, thanks for removing it. Although I have to ask you to not fix (set it to 2) this setting in this PR as it would generate too many out of context code for reviewing. Removing is great, thanks.